csm/conf/csm_cleanup.py

```diff
@@ -84,33 +84,40 @@ def process_stats(args):
     es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
     remove_old_indexes(es, args.d, args.n, args.e)
 
-def process_auditlogs(args):
+def process_es_cleanup(args):
     # Pass arguments to worker function
     # remove data older than given number of days
     es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    es.remove_old_data_from_indexes(args.d, args.n, args.i, args.f)
+    es.remove_old_data_from_indexes(args.d, args.n, ["csmauditlog"], "timestamp")
```

Comment:
Get the collection names from either `database.yaml` or `command` so that if someone changes the collection name in database.yaml this script doesn't need to be changed.

Reply:
database.yml only have entreis  of collectins and not the feilds. Specific feild has to be mentioned as deletion query runns on that feild. There is no common feild accross all the indexes.

Anyhow the script has to be changed in case there is addition of any index in future..